examples/v9/chrome/package.json

```diff
@@ -4,14 +4,14 @@
   "description": "running Cypress using Chrome",
   "main": "index.js",
   "scripts": {
-    "test": "cypress run",
+    "test": "cypress run --browser chrome",
```

Comment:
👍

Reply:
That change made sense when running locally under manual control. It is not relevant when [example-chrome.yml](https://github.com/cypress-io/github-action/blob/master/.github/workflows/example-chrome.yml) is running because either Electron is selected by default or `chrome` is explicitly selected in the workflow.